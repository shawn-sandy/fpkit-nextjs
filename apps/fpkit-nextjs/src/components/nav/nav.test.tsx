import React from "react";

import { describe, expect, test, it } from "vitest";
import { render, screen } from "@testing-library/react";

import Nav from "./nav";
describe("App renders correctly", () => {
  test("renders without crashing", () => {
    render(<Nav />);
    expect(screen).toMatchSnapshot();
  });
});
