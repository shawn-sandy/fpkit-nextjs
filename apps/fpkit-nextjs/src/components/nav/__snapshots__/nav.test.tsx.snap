// Vitest Snapshot v1

exports[`App renders correctly > renders without crashing 1`] = `<nav />`;
